.github/workflows/unit-and-integration-tests-unified.yml

```diff
@@ -20,7 +20,8 @@ on:
       - '**/*.md'
       - 'dev/**'
       - 'docs/**'
-      - 'examples/**'
+      - 'examples/**/*.md'
```

Comment:
this should already be covered by whats on line 20.

Reply:
good point, removed the redundant exclusion.